tests/unit/test_tenant.py

```diff
@@ -12,30 +11,22 @@ def setup(self):
         self.custom_404_msg = "Tenant not found"
         self.schema = TenantSchema
 
-    def test_only_active_lease_returned(self, faker, create_lease, create_tenant):
-        tenant = create_tenant()
-
-        # Create an expired lease for the tenant
-        lease_expired_end = faker.date_time_this_decade(
-            before_now=True, after_now=False
-        )
-        lease_expired_start = lease_expired_end - datetime.timedelta(days=365)
-        lease_expired = create_lease(
-            tenant=tenant,
-            dateTimeStart=lease_expired_start,
-            dateTimeEnd=lease_expired_end,
-        )
-
-        # Create an active lease for the tenant
-        lease_active = create_lease(
-            tenant=tenant, property=PropertyModel.find(lease_expired.propertyID)
-        )
-
-        # Active lease should be the only one that shows up
-        assert tenant.json()["lease"] == lease_active.json()
-
 
 @pytest.mark.usefixtures("empty_test_db")
-class TestTenantFactory:
-    def test_create_tenant(self, create_tenant):
-        assert create_tenant()
+class TestTenantModel:
+    def test_json(self, create_tenant, create_lease):
+        tenant = create_tenant()
+        lease = create_lease(tenant=tenant)
+
+        assert tenant.json() == {
+            "id": tenant.id,
+            "firstName": tenant.firstName,
+            "lastName": tenant.lastName,
+            "fullName": "{} {}".format(tenant.firstName, tenant.lastName),
```

Comment:
Not really related to this PR, but @NickSchimek when can we get rid of this ridiculous `fullName` field in the json? 😂

Reply:
I had no plans to get rid of it, but we certainly can and should. Someone needs to make an issue on the frontend to stop depending on it, and once that's completed then we can remove it.